frontend/src/components/banners/LicenseBanner.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <div
+        v-if="showBanner"
+        class="ff-banner ff-banner-warning"
+        data-el="banner-license-banner"
+    >
+        <span>
+            <ExclamationCircleIcon class="ff-icon mr-2" />
+            <span v-if="license.expiring">Your FlowForge license will expire in {{ license.daysRemaining }} day{{ license.daysRemaining > 1 ? 's': '' }}.</span>
+            <span v-else-if="license.grace">Your FlowForge license has expired and will stop working in  {{ license.graceDaysRemaining }} day{{ license.graceDaysRemaining > 1 ? 's': '' }}.</span>
```

Comment:
```suggestion
            <span v-else-if="license.grace">Your FlowForge license has expired and will stop working in  {{ license.graceDaysRemaining }} day{{ license.graceDaysRemaining > 1 ? 's': '' }}.</span>
```

If #1744 decides we don't suspend anything, then we don't talk about grace periods in the UI at all - the license is either valid, valid-expiring-soon, or invalid.

Reply:
removed